### DISCUSSION TITLE
SKU com /

### DISCUSSION THREAD

**Discussion by elderdosantos** [rodurma/forum-meli#19] [Geral]
Olá pessoal.

Tenho que consumir a api de dados fiscais [http://api.mercadolibre.com/items/fiscal_information/$SKU] porém o SKU a ser consultado tem uma / no meio dele.
Como vocês trataram caso assim? Tratar com urlencode() nao resolve, sempre retorna 404

**Comment by gersonfs** [comment]
Tentou adicionar contra barra?
Exemplo: 
`123\/123`
